contribute.html

```diff
@@ -48,12 +48,13 @@
 				<button class="navbar-toggler" data-toggle="collapse" data-target=".navbar-collapse"><svg viewBox="0 0 12 12"><path d="M1 1h10M1 6h10M1 11h10"/></svg></button>
 				<!-- Main navigation -->
 				<div class="navbar-collapse collapse">
-					<a class="nav-link" href="/">Home</a>
-					<a class="nav-link" href="/#features">Features</a>
-					<a class="nav-link" href="/#download">Download</a>
-					<a class="nav-link" href="/#gettingstarted">Getting Started</a>
-					<a class="nav-link" href="#contribute">Contribute</a>
-					<a class="nav-link" href="/#testimonials">Testimonials</a>
+					<a class="nav-link" href="#home">Home</a>
+					<a class="nav-link" href="#features">Features</a>
+					<a class="nav-link" href="#download">Download</a>
+					<a class="nav-link" href="#gettingstarted">Getting Started</a>
+					<a class="nav-link" href="#stats">Stats</a>
+					<a class="nav-link" href="contribute.html">Contribute</a>
+					<a class="nav-link" href="#testimonials">Testimonials</a>
```

Comment:
These these are different HTMLs, it must stay as it was with the leading slash (aside of the single contribute ID on this page). So you basically want to add the stats page to the navigation everywhere? We'll need to check whether it fits or not and whether the breaking point on mobile phones for switching to burger menu is still correct. Else, easiest would be to reduce the padding of those buttons a little to make it fit.

Reply:
Yes, my intention was to have the same navigation on every website page.

Reply:
When adding the stats page, I wasn't sure whether both (stats + contribute) will fit so left it out first, also as it is already linked via the first big button below the slider. We can try it and in case adjust padding, but as said, the other links need to stay untouched, else they point to non-existing elements 😉.

Reply:
Ok, could you correct this? I only can edit "blind" and have no website sandbox.

Reply:
Ah okay, jep will do it.